company_website/templates/main_page.haml

```diff
@@ -2,7 +2,10 @@
 {% load pipeline %}
 {% stylesheet 'main' %}
 
+.overlay
```

Comment:
What does this doing?

Reply:
When we are opening menu on mobile devices it is to overlay shadow for all the content under the menu and if we click on shadow content menu will hide.